fluid/PaddleCV/image_classification/README_ngraph.md

```diff
@@ -0,0 +1,43 @@
+
+# PaddlePaddle inference and training script
+This directory contains configuration and instructions to run the PaddlePaddle + nGraph for a local training and inference.
+
+# How to build PaddlePaddle framework with NGraph engine
+In order to build the PaddlePaddle + nGraph engine and run proper scripti,  follow up a few steps:
+1. build the PaddlePaddle project
+2. download pre-trained model data
+3. set env exports for nGraph and OMP
+5. run the inference/training script
+
+Curently supported models:
+* ResNet50 (inference and training).
+
+Short description of aforementioned steps:
+
+## 1. Build paddle
```

Comment:
we don't need to build paddle. Use `pip install paddlepaddle==1.4` is enough.

Reply:
Is `WITH_NGRAPH=ON` by default?